servers/plugins/org.fusesource.ide.server.karaf.core/src/org/fusesource/ide/server/karaf/core/server/subsystems/Karaf2xPublishController.java

```diff
@@ -213,12 +209,20 @@ public int transferBuiltModule(IModule[] module, IPath srcFile, IProgressMonitor
 
 	@Override
 	public int removeModule(IModule[] module, IProgressMonitor monitor) throws CoreException {
-		BundleDetails bd = new ModuleBundleVersionUtility().getBundleDetails(module, null);  // This will clearly break... wtf.  Crap. 
+		IPath tmpArchive = getTempBundlePath(module);
+		
+		BundleDetails bd = new ModuleBundleVersionUtility().getBundleDetails(module, tmpArchive);  // This will clearly break... wtf.  Crap. 
```

Comment:
it was working for pure OSGi when the version has no .qualifier. I implemented it to have it working with .qualifier too (by just not checking version in this case)